dts/arm/rpi_pico/rp2040.dtsi

```diff
@@ -10,9 +10,17 @@
 #include <zephyr/dt-bindings/clock/rpi_pico_clock.h>
 #include <zephyr/dt-bindings/i2c/i2c.h>
 #include <zephyr/dt-bindings/regulator/rpi_pico.h>
+#include <zephyr/dt-bindings/reset/rpi_pico_reset.h>
 #include <mem.h>
 
-#include "rpi_pico_common.dtsi"
+#include "override.dtsi"
```

Comment:
```suggestion
#include <arm/rpi_pico/override.dtsi>
```

Needs to be the full include path using `<` and `>` because it will scan include folders whereas `"` will only include the override file in the same folder. Rest is good